pom.xml

```diff
@@ -24,16 +25,18 @@
   </licenses>
 
   <scm>
-    <connection>scm:git:git://github.com/jenkinsci/build-timeout-plugin.git</connection>
-    <developerConnection>scm:git:git@github.com:jenkinsci/build-timeout-plugin.git</developerConnection>
-    <url>https://github.com/jenkinsci/build-timeout-plugin</url>
-    <tag>HEAD</tag>
+    <connection>scm:git:git://github.com/${gitHubRepo}.git</connection>
+    <developerConnection>scm:git:git@github.com:${gitHubRepo}.git</developerConnection>
+    <tag>${scmTag}</tag>
+    <url>https://github.com/${gitHubRepo}</url>
   </scm>
 
   <properties>
-    <!-- Dependency org.jenkins-ci.plugins:mailer:jar:412.v7deeda_155287 requires Jenkins 2.331 or higher. -->
-    <jenkins.version>2.334</jenkins.version>
-    <java.level>11</java.level>
+    <revision>1.21</revision>
+    <changelist>-SNAPSHOT</changelist>
+    <jenkins.version>2.332.1</jenkins.version>
+    <java.level>8</java.level>
```

Comment:
This is highest `java.level` currently supported by the Jenkins project build infrastructure.